README.md

```diff
@@ -12,7 +12,7 @@ In order to become a profitable, and sustainable organization, we need you to bu
 
 While you are building out the Bangazon Platform, your leadership team will ensure that the following foundational skills are developed by each team, and by each individual on the team.
 
-![Learning objectives](./learning-objectives.png)
+TODO: Replace with new visualization or list of learning objectives
```

Comment:
Has something changed in the learning objectives?

Reply:
The whole branch on SOLID Principles is not something we cover (other than single responsibility, lightly). And we don't expressly teach the pillars of OOP either. Students may learn some of these things as a result of the work we do, but they aren't "taught". Certain students would look at this image and assume it translates directly to the curriculum, which could lead to confusion/disappointment when they don't receive specific instruction on those objectives.

Reply:
We do teach OOP but I agree on the SOLID stuff. We can nix that